backend/tracim_backend/command/cleanup.py

```diff
@@ -0,0 +1,222 @@
+import argparse
+import typing
+
+from pyramid.scripting import AppEnvironment
+
+from tracim_backend import UserDoesNotExist
+from tracim_backend.command import AppContextCommand
+from tracim_backend.lib.cleanup.cleanup import CleanupLib
+from tracim_backend.lib.core.user import UserApi
+from tracim_backend.models.auth import User
+from tracim_backend.models.tracim_session import unprotected_content_revision
+
+
+class DeleteResultIds(object):
+    def __init__(self, user_id: int, workspace_ids: typing.List[int] = None):
+        self.user_id = user_id
+        self.workspace_ids = workspace_ids or []
+
+
+class DeleteUserCommand(AppContextCommand):
+    def get_description(self) -> str:
+        return """Remove user and associated information from database"""
+
+    def get_parser(self, prog_name: str) -> argparse.ArgumentParser:
+        parser = super().get_parser(prog_name)
+        parser.add_argument(
+            "-best-effort",
+            "--best-effort",
+            help="anonymise if needed user",
+            dest="best_effort",
+            default=False,
+            action="store_true",
+        )
+        parser.add_argument(
+            "-t",
+            "--test-mode",
+            help="test mode",
+            dest="test_mode",
+            default=False,
+            action="store_true",
+        )
+        parser.add_argument(
+            "-f",
+            "--force",
+            help="force user deletion",
+            dest="force",
+            default=False,
+            action="store_true",
+        )
+        parser.add_argument(
+            "-l", "--login", nargs="+", help="User logins (email)", dest="logins", required=True
+        )
+        return parser
+
+    def take_app_action(self, parsed_args: argparse.Namespace, app_context: AppEnvironment) -> None:
+        self._session = app_context["request"].dbsession
+        self._app_config = app_context["registry"].settings["CFG"]
+
+        if parsed_args.test_mode:
+            print("(!) Running in Test mode, not change will be applied.")
+        if parsed_args.force:
+            print("/!\\ Running in force mode, database created may be broken /!\\.")
```

Comment:
```suggestion
            print("/!\ Running in force mode, database created may be broken /!\.")
```

Or this is conventional maybe ?

Reply:
this is needed for flake8 to escape "\"

Reply:
ok